src/internal/operations/backup.go

```diff
@@ -407,6 +397,119 @@ func consumeBackupDataCollections(
 	return bu.BackupCollections(ctx, bases, cs, sel.PathService(), oc, tags)
 }
 
+func matchesReason(reasons []kopia.Reason, p path.Path) bool {
+	for _, reason := range reasons {
+		if p.ResourceOwner() == reason.ResourceOwner &&
+			p.Service() == reason.Service &&
+			p.Category() == reason.Category {
+			return true
+		}
+	}
+
+	return false
+}
+
+func mergeDetails(
+	ctx context.Context,
+	ms *store.Wrapper,
+	detailsStore detailsReader,
+	mans []*kopia.ManifestEntry,
+	toMerge map[string]path.Path,
```

Comment:
`toMerge` has, in this follow-up PR, revealed itself to be a bit ambiguous.  It describes the goal of the map, but not its contents.  For a dev who doesn't already grok the design at large, the intent is difficult to tie in to a mental model.  Something that more directly describes the value would make it more useful.

I honestly don't have a good replacement name off the top of my head.  `shortRefsFromPrevBackup`?